### PR TITLE
Cts create api

### PR DESCRIPTION
**Pull request by wilkermichael**
Some changes made by @lornasong are required for these documentation changes, ignore commits made by Lorna in this review, they are part of a [separate PR](https://github.com/hashicorp/consul/pull/12303).

[task create api preview link](https://consul-bzlloxvss-hashicorp.vercel.app/docs/nia/api/tasks#create-task)
[task create cli preview link](https://consul-bzlloxvss-hashicorp.vercel.app/docs/nia/cli/task#task-create)

### CODE REVIEW

**Review by lornasong — APPROVED** [review]
looks good to me! we chatted offline that something in the process is automatically updating the table formatting.

File: website/content/docs/nia/cli/task.mdx [hunk 1]
```diff
@@ -4,24 +4,86 @@ page_title: Task Command
...
+In addition to [general options](/docs/nia/cli/cli-overview#general-options) this command also supports the following:
+
+| Name         | Required | Type   | Description                                                                                                         |
+| ------------ | -------- | ------ | ------------------------------------------------------------------------------------------------------------------- |
+| `-task-file` | Required | string | The path to the task configuration file. See [configuration information](/docs/nia/configuration#task) for details. |
+
+### Example
+
+```shell-session
+$ consul-terraform-sync task create -task-file=task.hcl
```
[the remark this answers is not in the document]
Comment (wilkermichael) [reply website/content/docs/nia/cli/task.mdx:31]: Sure I can add one

File: website/content/docs/nia/api/tasks.mdx [hunk 2]
```diff
@@ -4,49 +4,223 @@ page_title: Consul-Terraform-Sync Tasks API
...
+| Name  | Required | Type   | Description                                                                                                                                                                                                                                                                                                                                         | Default |
+| ----- | -------- | ------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| `run` | Optional | string | Values can only be `"inspect"` and `"now"`.<br/><ul><li>`"inspect"`: Does not update the task but returns information on if/how resources would be changed for the proposed task update.</li><li> `"now"`: Updates the task accordingly and immediately runs the task, rather than allowing the task to run at the natural daemon cadence</li></ul> | none    |
```
[the remark this answers is not in the document]
Comment (wilkermichael) [reply website/content/docs/nia/api/tasks.mdx:101]: yep, stole this from the create description and missed changing this, I'll update (for lack of a better word) to create.

File: website/content/docs/nia/cli/task.mdx [hunk 3]
```diff
@@ -4,24 +4,86 @@ page_title: Task Command
...
+
+It is not to be used for updating a task and will not create a task if the task name already exists.
+
+### Usage
+
+`consul-terraform-sync task create [options] -task-file=<task config>`
+
+**Options:**
+
+In addition to [general options](/docs/nia/cli/cli-overview#general-options) this command also supports the following:
```
[the remark this answers is not in the document]
Comment (lornasong) [reply website/content/docs/nia/cli/task.mdx:22]: We chatted about this offline - @wilkermichael  pointed out that the `-task-file` is still a type of option (a required option). Arguments are like the task-name in `consul-terraform-sync delete task <task-name>`. Create Task CLI only has options and no arguments. My suggestion doesn't make sense, please disregard.

**Pull request merged**